From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aswin-mcw <aswin.kalies@multicorewareinc.com>
Date: Thu, 30 Jul 2026 16:49:10 +0530
Subject: [PATCH 095/109] fixup! ci: run Cygwin's testsuite on the MSYS2
 runtime

Download the worktree artifact into the workspace root instead of a
directory named wt.tgz, so that 'tar -xzf wt.tgz' unpacks the tarball
rather than failing on a directory of the same name.

(cherry picked from commit 1e1df1d3cc131b0a8ddd8c04d3a91b3ad262d761)
(cherry picked from commit 62dc7e6f69ffd7104fde27175c2d6a2cbfc2cf1b)
---
 .github/workflows/build.yaml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/.github/workflows/build.yaml b/.github/workflows/build.yaml
index fe50f7b75..6d0474475 100644
--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -135,7 +135,6 @@ jobs:
         uses: actions/download-artifact@v7
         with:
           name: wt.tgz
-          path: wt.tgz
 
       - name: Extract worktree
         shell: msys2 {0}
